src/skin/tooltips.cpp

```diff
@@ -396,6 +396,18 @@ void Tooltips::addStandardTooltips() {
             << tempoDisplay
             << tr("When tapped repeatedly, adjusts the BPM to match the tapped BPM.");
 
+    add("shift_cues_earlier")
+            << tr("Shift cues earlier")
+            << tr("Shift cues imported from Serato or Rekordbox if they are slightly off time.")
+            << tr("Left click: shift 1 millisecond earlier")
```

Comment:
These descriptions are wrong. The two constants are 10 and 1 milliseconds respectively.